Hyperdrive/HyperBlueprint.swift

```diff
@@ -444,6 +452,11 @@ public class HyperBlueprint : Hyperdrive {
 
       if let relation = action.relation {
         let transition = HTTPTransition.from(resource:resource, action:action, URL:actionURI)
+        if let allowedMethods = allowedMethods {
```

Comment:
No problem, there's no such thing as a bad question!

This is a common pattern in Swift to use the same variable name, since you're stating that that variable is non-optional within the if case. Where you have access to the non optional type, for example `String` instead of an optional string type `String?`.

Here's a little exerpt from the [Swift book](https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/TheBasics.html) if you're interested in [learning more](https://blog.sabintsev.com/optionals-in-swift-c94fd231e7a4):

<img width="678" alt="screen shot 2015-07-10 at 11 00 29" src="https://cloud.githubusercontent.com/assets/44164/8625319/e553021a-26f2-11e5-99f1-9f0a9387da86.png">

<img width="701" alt="screen shot 2015-07-10 at 10 59 16" src="https://cloud.githubusercontent.com/assets/44164/8625294/c37d0028-26f2-11e5-9502-2a9df3384a6b.png">